media/components/sdfv/index.html

```diff
@@ -114,6 +114,29 @@ <h5 id="info-title"></h5>
     <script src="{{ CSP_SRC }}/components/sdfv/analysis/analysis.js"></script>
     <script src="{{ CSP_SRC }}/components/sdfv/properties/properties.js"></script>
     <script src="{{ CSP_SRC }}/components/sdfv/transformation/transformation.js"></script>
+
+    <!-- Load in the monaco editor. -->
+    <script>
```

Comment:
really? :)
(maybe we should comment it out for now)

Reply:
on second thought, maybe it's better to leave uncommented, to see if someone complains

Reply:
Yeah I opted for leaving it in and uncommented because I plan on adding the actual monaco editor in within the next few days if possible - commented or not, in the end both works for me :)